### DISCUSSION TITLE
Fixed price for x amount of hours

### DISCUSSION THREAD

**Discussion by danieldrinhausen** [kimai/kimai#4825] [General]
Hey guys,

trying to figure out how I can do the following.

I have a fixed price project, let's say 1000 Euro for 10 hours of work as an example. I want to bill the customer the 1000 Euro straight away but also want to add the hours I have worked on this project (Maybe over many month), not billable then to Kimai. How can I do that?

Only thing I can think of is having one project with two activities, one is fixed priced (1000) where I have to put one entry with lets say one hour and the other activity would be not billable which I can use to track how much time I have spent already.

Any smarter way?

Thanks

**Comment by danieldrinhausen** [comment]
Have found a workaround. Basically having the first time tracked record a fixed amount and setting it afterwards to non billable.

**Comment by kevinpapst** [comment]
Use the ExpensesBundle - thats made for "fixed prices":
https://www.kimai.org/store/expenses-bundle.html

And buying it supports further development of Kimai
